### DISCUSSION TITLE
Show `{key}: {value}` next to `KeyValuePair` structs in 3D object inspector and Raw Message panel

### DISCUSSION THREAD

**Discussion by jhurliman** [foxglove/community#695] [Q&A]
The Foxglove schema for scene entities supports attaching arbitrary metadata to scene entities with a struct that contains `key` and `value` fields, but visualizing this data is awkward because you need to open the metadata structure in JSONView then open each individual entry to see the keys and values.

**Comment by amacneil** [comment]
We're working on it!

Cross linking https://github.com/foxglove/studio/pull/5701
